docs/application/web/guides/device/system-information.md

```diff
@@ -234,6 +234,31 @@ To receive notifications on property value changes:
    var id = tizen.systeminfo.addPropertyValueArrayChangeListener('SIM', successCallback);
    ```
 
+
+<b>Remark:</b>
+
+In case of `WIFI_NETWORK` property, value change listener is triggered on `ipAddress` and `ip6Address` properties change (the network layer). These changes could be not consistent with physical layer (`status` or `signalStrength` properties of physical adapter).
```

Comment:
Please advice me here, the following is my understanding:

In the case of `WIFI_NETWORK` property, value change listener is triggered on `ipAddress` and `ip6Address` property change in the network layer. These changes are not consistent in the `status` or `signalStrength` properties of a physical adapter in the physical layer.

Has the rephrase alter the original meaning of the text?